comp/core/autodiscovery/listeners/snmp.go

```diff
@@ -462,3 +507,7 @@ func convertToCommaSepTags(tags []string) string {
 	}
 	return strings.Join(normalizedTags, tagSeparator)
 }
+
+func getSubnetVarKey(subnet snmpSubnet) string {
```

Comment:
This should leave in a shared file with the struct type declaration

Reply:
agree, 750a647a413a09201987783559f71f01303b0ac7